library/Fission/Web/Swagger.hs

```diff
@@ -39,10 +38,10 @@ docs host' =
     |> user
 
 app :: HasSwagger api => Proxy api -> Web.Host -> Swagger
-app proxy (Web.Host (BaseUrl { baseUrlHost })) =
+app proxy appHost =
   proxy
     |> toSwagger
-    |> host               ?~ Host baseUrlHost Nothing
+    |> host               ?~ Host (Web.getHostName appHost) Nothing
```

Comment:
Does that unwrap deeply enough?

EDIT

Oh, I see. There's a helper function. That confused me on first pass, becasue the name is so similar. I'm not sure if `master` changed under you (uses pattern matching) 🤔 Another version could be:

```haskell
host ?~ Host (appHost |> Web.getHostName |> baseUrlHost) Nothing
```

or rename

```haskell
getRawHost :: Host -> String
```